tests/core/engine_adapter/docker/clickhouse/full.xml

```diff
@@ -0,0 +1,1690 @@
+<!-- TODO: I dumped this from the docker image so I could see the full extent of the default config to decide what needed to be overridden -->
+<!-- It needs to be removed at some point -->
```

Comment:
By at some point, do you mean before merging this PR or sometime in the future?

Reply:
@erindru

Reply:
Mostly asking because if we plan to merge this in I don't really even know from the message what needs to be changed in order to remove this.

Reply:
Yeah this can be removed now, I was just referring to it in order to create the other xml files